docs/howto/rts.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: advanced_rts
+id: howto_rts
```

Comment:
Let's get rid of this one (it's empty). We can add it back at the right place when someone writes it.